legal-api/src/legal_api/services/authz.py

```diff
@@ -164,24 +166,21 @@ def has_roles(jwt: JwtManager, roles: List[str]) -> bool:
                 }
             },
             'courtOrder': {
-                'legalTypes': ['SP', 'GP', 'CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'],
-                'blockerChecks': {
-                    'business': [BusinessBlocker.DEFAULT]
-                }
+                'legalTypes': ['SP', 'GP', 'CP', 'BC', 'BEN', 'CC', 'ULC']
             },
             'dissolution': {
                 'voluntary': {
-                    'legalTypes': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC', 'SP', 'GP'],
+                    'legalTypes': ['CP', 'BC', 'BEN', 'CC', 'ULC'],
                     'blockerChecks': {
                         'warningTypes': [WarningType.MISSING_REQUIRED_BUSINESS_INFO],
-                        'business': [BusinessBlocker.DEFAULT]
+                        'business': [BusinessBlocker.DEFAULT, BusinessBlocker.GOOD_STANDING]
                     }
                 },
                 'administrative': {
-                    'legalTypes': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC', 'SP', 'GP'],
+                    'legalTypes': ['CP', 'BC', 'BEN', 'CC', 'ULC'],
                     'blockerChecks': {
                         'warningTypes': [WarningType.MISSING_REQUIRED_BUSINESS_INFO],
-                        'business': [BusinessBlocker.DEFAULT]
+                        'business': [BusinessBlocker.DRAFT_PENDING]
```

Comment:
Does this prevent a staff user from starting an Administrative Dissolution if the business is already historical?

Reply:
Oh, this code is in the ACTIVE object, isn't it?